src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -103,8 +103,9 @@ export class ConsoleMenu extends Component<Props, State> {
 
   render() {
     const button = (
-      <EuiButtonIcon
-        iconType="wrench"
+      <button
+        className="euiButtonIcon--primary"
+        style={{ lineHeight: 'inherit' }}
         onClick={this.onButtonClick}
         data-test-subj="toggleConsoleMenu"
         // @ts-ignore
```

Comment:
Unrelated to this PR, but you can use `i18n.translate()` here to return a string and remove the `//@ts-ignore`.

(Currently it's being rendered as: `aria-label="[object Object]" `)

Reply:
Will do!